examples/inference/benchmark_vllm.py

```diff
@@ -0,0 +1,92 @@
+"""Benchmark the latency of processing a single batch of requests."""
+"""Modified from https://github.com/vllm-project/vllm/blob/main/benchmarks/benchmark_latency.py
+The only change is printing the throughput.
+"""
+import argparse
+import time
+
+import numpy as np
+import torch
+from tqdm import tqdm
+from vllm import LLM, SamplingParams
+
+
+def main(args: argparse.Namespace):
+    print(args)
```

Comment:
remove this or give it a better print format.

Reply:
done